tools/cli/src/test/java/org/apache/carbondata/tool/CarbonCliTest.java

```diff
@@ -275,9 +277,14 @@ public void testSummaryAllColumns() {
   @Test
   public void testSummaryAllColumnsForOneFile() {
     CarbonFile folder = FileFactory.getCarbonFile(path);
-    CarbonFile[] carbonFiles =
-        folder.listFiles(file -> file.getName().endsWith(CarbonTablePath.CARBON_DATA_EXT));
-    String[] args = { "-cmd", "summary", "-p", carbonFiles[0].getCanonicalPath(), "-C" };
+    CarbonFile[] allFiles = folder.listFiles();
```

Comment:
use File filters to get only data files...No need to filter

Reply:
@lamber-ken according to the old testcase we only need .carbondata files.

Reply:
> @lamber-ken according to the old testcase we only need .carbondata files.

right, so it needs filter

Reply:
I asked manish to pass a carbonfilter to the list files interface instead of looping and filtering

Reply:
> I asked manish to pass a carbonfilter to the list files interface instead of looping and filtering

ok, it is just a misunderstanding.